examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -189,23 +193,22 @@ class _ExpansionPanelsDemoState extends State<ExpasionPanelsDemo> {
             });
           }
 
-          return new CollapsibleBody(
-            margin: const EdgeInsets.symmetric(horizontal: 16.0),
-            child: new Form(
+          return new Form(
+            key: form1Key,
+            child: new CollapsibleBody(
+              margin: const EdgeInsets.symmetric(horizontal: 16.0),
+              onSave: () { form1Key.currentState.save(); close(); },
```

Comment:
you're in the form here, so you could avoid using a Key if you instead put a Builder below the form and used Form.of